pandas/tests/frame/test_dtypes.py

```diff
@@ -850,6 +850,35 @@ def test_arg_for_errors_in_astype(self):
 
         df.astype(np.int8, errors='ignore')
 
+    def test_arg_for_errors_in_astype_dictlist(self):
+        """Test that the errors=ignore argument silences the exception that is
```

Comment:
We don't use docstrings for tests. Just a comment for `# GH 25905` should suffice

Reply:
Removed in ce115a8ff91207f3c29f021421ca8abfa61190a5